docs/specs/genesis.md

```diff
@@ -16,7 +16,8 @@ NOTE: This does not (yet) specify the application state (e.g. initial distributi
 
 ### Sample genesis.json
 
-This example is from the Basecoin mintnet example ([link to file](https://github.com/tendermint/mintnet/blob/master/examples/basecoin/mach1/core/genesis.json)).
+This example is from the Basecoin mintnet example:
```

Comment:
need to fix this since mintnet no longer a thing. should have a way to make testnet genesis in tendermint/tools I think